content/cpi-api-v2-method/info.md

```diff
@@ -0,0 +1,44 @@
+# info
+
+Returns information about the CPI to help the Director to make decisions on which CPI to call for certain operations in a multi CPI scenario.
+
+
+## Arguments
+
+No arguments
+
+
+## Result
+
+ * `stemcell_formats` [Array of strings]: Stemcell formats supported by the CPI. Currently used in combination with `create_stemcell` by the Director to determine which CPI to call when uploading a stemcell.
```

Comment:
Could we add how this relates to the stemcell metadata? Do we already have some documentation explaning when which CPI will be selected?

Reply:
We have updated the info pages to remove the mention about combinations and things. Instead, we have added that information to the `build-stemcell` page. Which references back to the CPI v2 info page.